src/main/java/lv/ctco/javaschool/vote/entity/EventDto.java

```diff
@@ -7,20 +7,9 @@
 import java.util.List;
 
 public class EventDto {
```

Comment:
I suggest placing DTO classes into a separate package, such as `lv.ctco.javaschool.vote.dto`.

Reply:
Dto classes are placed in separate package.